friends/src/Components/Form/Form.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+const Form = props => {
+    return (
+        <div>
+         <form>
+            <label for="name">Name:</label>
+            <input type="text" name="name" onChange={props.inputHandler}/>
```

Comment:
Each of these inputs needs a `value` attribute, which you could get by passing in the `friend` object (on state in App.js) into the Form component as props. Then assign the value of each input to the respective properties on the object.

`props.inputHandler` should also be set to `props.value` because that's what you're calling the `inputHandler` method on line 54 in App.js